tableauserverclient/namespace.py

```diff
@@ -1 +1,28 @@
-NAMESPACE = {'t': 'http://tableau.com/api'}
+OLD_NAMESPACE = 'http://tableausoftware.com/api'
+NEW_NAMESPACE = 'http://tableau.com/api'
+
+class Namespace(object):
+    def __init__(self):
+        self._namespace = {'t': NEW_NAMESPACE}
+        self._detected = False
+
+    def __call__(self):
+        return self._namespace
+
+    def detect(self, xml):
+        if self._detected:
+            return
+
+        lines = (n for n in xml.split('\n') if n.startswith('<tsResponse'))
```

Comment:
There's breaks all ova' them tests because of str/bytes stuff.

I think we can reduce the parsing overhead a bit as well, by parsing the namespace outta the root element.

Something like this:
```
>>> root = tree.getroot()
>>> root.tag
'{http://maven.apache.org/POM/4.0.0}project'
```

And then run the find over just that smaller string

Reply:
hrm.  Weird, I even tested wth python 3.6 on my mac.  yeah I think I like this solution better.